README.md

```diff
@@ -1,16 +1,16 @@
-## Getting Started (Contributors Only)
+# Getting Started (Contributors Only)
 
 1. Clone this repo and the frontend repo (cdc-frontend)
 2. Grab the secrets
 3. todo: continued steps to get project set up locally
 
 ---
 
-## About Calgary Distress Center
+# About Calgary Distress Center
 
 todo
 
-### Team
+# Team
```

Comment:
Any reason for making Team/about/getting started all on the same level (and using the bigger header)? It's just inconsistent with frontend now. Not a big deal I guess

Reply:
The .md format only seems to support nesting to 4 `####` and for running the backend repo topic, I wanted to subdivide the sections. But with the current file, I wasn't able to do that. Then I noticed that we started at 2 `##` rather than 1 `#`, so I moved everything up 1 header.

Reply:
Makes sense. The contrast from `#` to paragraph text just looks so bad for me 😭  but it's okay